### DISCUSSION TITLE
2024 Akash Community Support Proposal

### DISCUSSION THREAD

**Discussion by CaptainSplince** [akash-network/community#394] [Governance Proposals]
# Overview

The Akash community is vast. It spans across geographic regions and social media channels. This is a good thing! The more people who follow Akash activities, the more likely they introduce new community members to the ecosystem. However, with the ever-growing size of the Akash community, the ability for community support, engagement, and education becomes difficult at scale. This proposal aims to alleviate community growth pains by providing Akash community members with improved support, resources and training.

## 1. Insiders & Vanguards Program

The Akash Insider’s program launched with much fanfare in late 2021. The initial concept of the program was to find community members who have either been actively engaged in Akash-related activities or onboard community members who have shown interest in participating in Akash activities. We initially onboarded 10 Insiders. Since then it has grown to 50 across the globe.

The Akash Vanguards program launched in mid-2022. Vanguards are the Akash community's most engaged members. It is a small group of highly active and supportive community members. Currently, we have 9 Akash Vanguards and are forecasting the need for an additional 5 Vanguards in 2024.

### Insider Responsibilities

There are no requirements to join the Insiders community. It is completely volunteer-based with optional activities. That being said, there is one unwritten rule for joining the program. It is expected that if you apply and join, you will be fairly active in community engagements. Examples but not limited to - Share feedback, highlight issues, join calls, provide community support.

Interested community members can apply to join [here.](https://akash.network/community/insiders/)

### Vanguard Responsibilities

The Akash Vanguards program has requirements to join. However, requirements are incentivized. A full overview of the program can be found [here](https://docs.google.com/document/d/1IMOLARSNg5tnIsfQ4B5mfDrmx-MDJNpIFbLxFLuldas/edit).

### Insider Rewards

Since there are no requirements to join the program, there are no standardized rewards, incentives, or a salary for joining the program. However, there are ad hoc activities that are optional to get involved in that can have rewards attached. Ex: Testnet support, blog bounties, Akash feature updates, etc.

### Vanguard Rewards

Since the Vanguard’s program has requirements attached to it, there are rewards given to members. Full details can be found on the [Vanguards Overview](https://docs.google.com/document/d/1IMOLARSNg5tnIsfQ4B5mfDrmx-MDJNpIFbLxFLuldas/edit).

## 2. Community Support Bounties and Project Board

The 2023 Community Support Proposal allocated $5,000 monthly for community bounties. Although we've managed these efforts on an ad-hoc basis until now, we recognize the need for a more structured approach moving forward. The core team initially introduced these bounties to quickly support smaller-scale contributions that may not be suited for traditional governance proposals. Our goal was to make funding more accessible and responsive to community needs. As a reminder, all expenditures are captured on-chain and can be viewed on the Community Support Dashboard.

To enhance transparency and coordination, all Community activities, including bounties, will be visible on the Akash Community + Marketing Initiatives board. This board will enable everyone to track ongoing and upcoming initiatives easily. Board will be made public once this proposal passes.

Interested in seeing which initiatives have funding available? Simply navigate to the “Bounty” view on the board and select the $$ Bounty $$ Label to see what can be earned.

We value your innovative ideas. If you have an idea that could benefit the network, please share it with our Working Groups/Special Interest Groups[here](https://akash.network/community/core-groups/cg-list/) or discuss it with the core team and community members on Discord. We're ready to explore how these bounty funds can support your initiatives.

Please note, the core team is managing the project board. We'll prioritize bounties based on network needs and ensure submissions meet our criteria. To align your efforts with our bounty standards, we encourage you to consult with the core team or other community leaders for guidance or clarification.

## 2024 Community Support Budget

The full overview of community funds requested to support this program can be found [here](https://docs.google.com/spreadsheets/d/1hT_KUzAsJn6CqFFJdtdnMQeLvtgfHWpyTlmPV2E_oSI/edit).

### 2024 Community Support Budget TLDR:

- Vanguards: $168,000
- Community Rewards/Bounties/Support: $60,000

#### Quarterly Breakdown:

Budget will be taken out and proposed quarterly in new governance proposals. Each quarter budget will be reviewed and proposed given community fund resources and market conditions. Overclock labs will be the custodian of the funds. Any funds not used will be returned to the community pool.

- Q1: $57,000
- Q2: $57,000
- Q3: $57,000
- Q4: $57,000

**Total 2024 Community Support Budget: $228,000 USD paid in AKT**

## Next steps:

Due date for community feedback is *12/11/23* at that time this signaling proposal will be updated with feedback given and then will go live on chain.

**Comment by rodri-r** [comment]
This is a very well redacted proposal and in my opinion necessary for the network. Both the Insiders program and Vanguards program benefit Akash in different ways. 

I have seen both grow and improve over the last couple of years and I am sure we can make them even better.

**Reply by CaptainSplince** [reply]
Thanks @rodri-r - that's the plan. We learned a lot in 2023 and all those learnings have gone into this propoal.

**Comment by okhaimie-dev** [comment]
I support this proposal. Well structured and clearly explains the goals, and programs. Looking forward to 2024, and to see how the community expands.

**Comment by Aihonaken** [comment]
Love the transparency, and that we are going to have the community board to share a see the bounties people can join or submit to. Looking on it as additional tool for marketing to get more active people from community.  Also love that unused fund are returning to the pool, as it was in previous proposal. i support this.

**Reply by CaptainSplince** [reply]
Thank you @Aihonaken! 2024 is gunna be a BIG year for Akash!

**Comment by ronykris** [comment]
Crisp proposal!  100% support for this proposal. 2024 could be the year for compute, therefore it makes all the more sense to go full throttle on community building.

**Reply by CaptainSplince** [reply]
Thanks @ronykris!

**Comment by piber-dev** [comment]
i support this proposal, being able to help the community through support, guides, and more is key to onboard users from all different skill levels. i am already a vanguard, so i am not affected by this proposal compared to the previous one. i would love to help work with you and the other vanguards to onboard the right people, the community members that we can trust to be as helpful as possible 🙏

**Reply by CaptainSplince** [reply]
I've said it before, I believe the Akash community is one of the strongest in the industry and this proposal will help support and scale it's growth!

**Comment by hewi333** [comment]
**End of Q2 Update**

Q3 Proposal is [Live](https://www.mintscan.io/akash/proposals/259).
Community Support [Dashboard](https://lookerstudio.google.com/u/1/reporting/f12da20a-b5bf-4f80-8a45-ff8861a1473e/page/BIUND).

Total requirement for Q3 2024: $57,000 USD
Funds remaining on the wallet: 4,013.32 AKT
Total request for Q3: 16,343.82 AKT at the time of this proposal

*Note there was no proposal for Q2 due to excess funds.

**Reply by hewi333** [reply]
**End of Q3 Update**
 
Q4 Proposal is [Live](https://www.mintscan.io/akash/proposals/269). 
Community Support [Dashboard](https://lookerstudio.google.com/u/1/reporting/f12da20a-b5bf-4f80-8a45-ff8861a1473e/page/BIUND).

**Note:** Due to growing participation of contributions, we have increased budget for bounties to $10k per month from $5k. 
 
Total requirement for Q4 2024: $72,000 USD
 Funds remaining on the wallet: 1,704.11 AKT
 Total request for Q4: 21,902.45 AKT at the time of this proposal

**Comment by Arcmosis** [comment]
I support $AKT and vote yes https://www.mintscan.io/akash/tx/C640775838F6A89ECC312DFBA695BC3062B9F2FCB40B9F3B4DEDF26704423E3E?height=16912259

**Comment by Antropocosmist** [comment]
As a responsible validator, we can't support this proposal, even if we always glad to support proposals to develop the community, and this is why:

1. This is what we can see by the budget link:
![image](https://github.com/user-attachments/assets/2966fd25-d641-4622-a37a-d13fd30e55bb)

friends, after this we have no understanding for what and how you will use this funds

2. This is what we get after " please share it with our Working Groups/Special Interest Groups:
![image](https://github.com/user-attachments/assets/47a14243-deaa-4b97-ab0f-536d85c18db6)

Page just not found!

3. information from Vanguard’s program:
![image](https://github.com/user-attachments/assets/77cebef8-de22-47ea-a081-de524e291698)

We don't understand what is "an invite only, elite selection of Insiders"
Who will decide it? 

And what's about people who are contributed for Akash community development for a long time without any rewards?

4. After you click on link "Interested community members can apply to join [here.](https://akash.network/community/insiders/): you will get this:
![image](https://github.com/user-attachments/assets/f371946b-8e79-4191-a51b-06d229c59245)

**Reply by brewsterdrinkwater** [reply]
Thank you so much for the due diligence and for sharing your thoughts. It is much appreciated. 

The Akash Network website has been going through continued major updates over the past few months, as contributions have continued to increase. As a result the structure of pages has been changing. The working group for the Akash Website and Docs (wg-website) has not properly implemented redirects. That is not acceptable, and the practice will be discussed ahead of the next meeting. 

**1.** I will not speak in great detail to the budget sheet, but I know there are multiple tabs, that may provide some more detail. This is a program that Overclock Labs have been leading for over a year with great success and transparency. 

**2.** The proper links for Special Interest Groups (SIGs) and Working Groups (WGs) is right here: https://akash.network/development/welcome/

 a. On this page, you can see the full schedule for SIGs and WGs. This is the best way for contributors to get involved, introduce themselves to others in the Akash community, ask questions, etc. Many of the efforts supported by the Community Support Proposal comes from this framework. Again, these pages are continuing to be improved to make it more clear for new and existing Akash community members. While not required, there are many teams and invididuals that participate in the open source framework. 

 b. There is a bounty board here: https://github.com/orgs/akash-network/projects/6/views/2?sliceBy%5Bvalue%5D=%24%24+Bounty+%24%24 . Here is where you can track many of the efforts that come out of SIGs and WGs. 

 c. There are also community members and teams that are contributing to Akash Console. The contributions have allowed the core team to increase its velocity with larger features like trial wallets and fiat payments. See ways to contribute to Akash Console here: https://github.com/akash-network/console/issues

 d. Lastly, the Community Support funds are used to activate community members that are building POCs on Akash. Again, these ideas are discussed in public, and there is a clear goal for funds used. Efforts like wg-zealy started out as POCs funded by the Community Support Proposal. 
 
 **3.** Vanguards is a program that has been heavily discussed in sig-community and other SIGs for well over a year. The purpose of the Vanguards is to provide 24 / 7 support in Akash Discord. Discord is where the building is happening, and having an active Discord community where members can get answers quickly goes a long way. Vanguards program is currently managed by the Overclock Labs team, but there is constant collaboration with community on framework and responsibilities. More information needs to be shared on the website about the Akash Insiders and Vanguards. The wg-akash-website time is creating issues right now, to address the issues that you addressed. 

For those that have been contributing to Akash without rewards, THANK YOU! The goal with the Akash open framework has always been to create mechanisms for contributors to get paid, especially if they are making open source contributions that are benefitting the network. Community Support is just one mechanism to support contributors. Others include Akash Foundation Delegations, as well as the community pool where there are clear best practices on how to request funds from the Akash Community Pool.

**Comment by SophoclesSummer** [comment]
This proposal was approved on 1/10/2024 by a 92% majority. 
https://www.mintscan.io/akash/proposals/243
